recipes/mesmer/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "mesmer" %}
+{% set pypiname = "mesmer-emulator" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypiname[0] }}/{{ pypiname }}/{{ pypiname }}-{{ version }}.tar.gz
+  sha256: 529ec371032fa003b1af81a1dc608c7b86732610b84b51cdc475b902951c248d
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - dask
+    - distributed
+    - geopy
+    - numpy
+    - pandas
+    - scikit-learn
+    - statsmodels
+    - regionmask
+    - xarray
+
+test:
+  imports:
+    - mesmer
+  requires:
+    - pip
+  commands:
+    - pip check
+    - python -c "import mesmer; assert mesmer.__version__ == '{{ version }}', mesmer.__version__"
+
+about:
+  home: https://github.com/openscm/mesmer
```

Should this be:
```suggestion
  home: https://github.com/MESMER-group/mesmer
```
? The other one 404's.